WardenApi/src/main/java/com/salesforce/dva/warden/dto/Policy.java

```diff
@@ -1,5 +1,165 @@
 package com.salesforce.dva.warden.dto;
 
-public class Policy {
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
 
+
+import com.salesforce.dva.warden.entity.SuspensionLevel;
+
+/**
+ * Policy Dto.
+ *
+ * @author  Ruofan Zhang (rzhang@salesforce.com)
+ */
+@SuppressWarnings("serial")
+//@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
You'll need to include faster jackson json as a dependency for this to resolve.  It shouldn't be commented out.